src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +98,73 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     * @param string|null $locale
+     * @param int|null    $maxRows
+     * @param int|null    $startRow
+     * @return AddressCollection
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = null, int $startRow = null): AddressCollection
```

Comment:
Maybe we should use better defaults?

`getCountryInfo(string $country = null, string $locale = null, int $maxRows = 10, int $startRow = 0)`